src/main/java/com/github/lunatrius/schematica/client/gui/save/GuiSchematicSave.java

```diff
@@ -48,10 +59,12 @@
 
     public GuiSchematicSave(final GuiScreen guiScreen) {
         super(guiScreen);
+        this.format = nextFormat();
     }
 
     @Override
     public void initGui() {
+        SchematicFormat.FORMAT_DEFAULT = Names.NBT.FORMAT_ALPHA;
```

Comment:
Is there a specific reason why the default format has to be reset back to alpha here?